Extension/src/common.ts

```diff
@@ -1300,6 +1300,31 @@ export function isValidIdentifier(candidate: string): boolean {
     return true;
 }
 
+export function getBaseStoragePath() {
+    let defaultCachePath: string = "";
+    let pathEnvironmentVariable: string | undefined = "";
+    switch (os.platform()) {
+        case 'win32':
+            defaultCachePath = "\\Microsoft\\vscode-cpptools\\";
+            pathEnvironmentVariable = process.env["LOCALAPPDATA"];
+            break;
+        case 'darwin':
+            defaultCachePath = "\\Library\\Caches\\vscode-cpptools\\";
```

Comment:
It seems odd to use backslashes for paths on linux and mac.  Can you flip these to forward slashes?